templates/download/risc-v/index.html

```diff
@@ -2,73 +2,129 @@
 
 {% block title %}下载用于RISC-V平台的Ubuntu{% endblock %}
 
-{% block meta_description %}在RISC-V平台上使用Ubuntu以获得相似的开发者体验，RISC-V开发板，RISC-V Ubuntu，RISC-V OS，RISC-V操作系统。{% endblock meta_description %}
+{% block meta_description %}
+  在RISC-V平台上使用Ubuntu以获得相似的开发者体验，RISC-V开发板，RISC-V Ubuntu，RISC-V OS，RISC-V操作系统。
+{% endblock meta_description %}
 
-{% block meta_copydoc%}https://docs.google.com/document/d/1kkJeq9HdH3iWV35HwihcJd3CxZw9Jy21W66lJl8VbWs/edit#{% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1SPeefGBJBh99XWEx0newZHPxp56mZuaElq1YtUzfY4M/edit#
+{% endblock meta_copydoc %}
 
 {% block content %}
-
-{% include "download/secondary-nav.html" %}
-
-<section class="p-strip--suru-topped">
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h1>下载适用于RISC-V平台的Ubuntu</h1>
-      <h2 class="p-heading--3">Ubuntu—现已支持多个RISC-V平台</h2>
-      <p>在您喜爱的RISC-V开发板上运行Ubuntu。Ubuntu近期在Ubuntu 22.04.2中已支持SiFive Unmatched，赛昉（StarFive） VisionFive和全志（Allwinner）哪吒开发板，在下方可下载对应开发板的镜像。</p>
-      <p>选择与您的硬件相匹配的操作系统镜像，将系统镜像刷入到 SD/microSD 卡上，再将其加载到您的板上即可开始体验Ubuntu。</p>
-      <p>
-        <a href="/contact" class="p-button">联系我们</a>
-      </p>
-    </div>
-    <div class="col-6 u-hide--medium u-hide--small u-align--center">
+  {% include "download/secondary-nav.html" %}
+  <section class="p-strip--suru-topped">
+    <div class="row u-vertically-center">
+      <div class="col-6">
+        <h1>下载适用于RISC-V平台的Ubuntu</h1>
+        <h2 class="p-heading--3">Ubuntu——现已支持多个RISC-V平台</h2>
+        <p>在您喜爱的RISC-V开发板上运行Ubuntu。</p>
+        <p>
+          选择与您的硬件相匹配的操作系统镜像，将系统镜像刷入到 SD/microSD 卡上，再将其加载到您的板上即可开始体验Ubuntu。
+          <br />
+          所有镜像都是基于64位Ubuntu服务器版而为开发者预览所打造。
```

Comment:
Looking at the copy doc, this should be a separate paragraph, not a break.

```suggestion
          <p>所有镜像都是基于64位Ubuntu服务器版而为开发者预览所打造。</p>
```